packages/mutation-testing-elements/src/style/bootstrap/_darkly-variables.scss

```diff
@@ -5,186 +5,187 @@
 // Color system
 //
 
-$darkly-white:    #fff !default;
```

Comment:
I don't mind the new formatting, but maybe we need to introduce prettier as a lint task to make sure we keep this consistent

Reply:
Yeah I can add that in a separate PR (after) to keep the diff for this PR smaller